docs/configuration/config-file-reference.md

```diff
@@ -2807,14 +2807,89 @@ bucket_store:
     # CLI flag: -experimental.tsdb.bucket-store.chunks-cache.max-get-range-requests
     [max_get_range_requests: <int> | default = 3]
 
-    # TTL for caching object size for chunks.
-    # CLI flag: -experimental.tsdb.bucket-store.chunks-cache.object-size-ttl
-    [object_size_ttl: <duration> | default = 24h]
+    # TTL for caching object attributes for chunks.
+    # CLI flag: -experimental.tsdb.bucket-store.chunks-cache.attributes-ttl
+    [attributes_ttl: <duration> | default = 24h]
```

Comment:
I see this (flag and config change) did not make into the CHANGELOG, is it intentional?

Reply:
No, it's an omission and changelog entry would be nice. On the other hand, old attribute name was not yet `released` (it's only on master), so probably not worth it?